lib/topify.js

```diff
@@ -77,19 +77,33 @@ function _macTopify(winPtrStr) {
 function _nixTopify(winPtrStr) {
   // TODO: we might need to further transform win before attempting to do this.
   // see https://developer.mozilla.org/en-US/Add-ons/Code_snippets/Finding_Window_Handles#Unix
+  
+  const gdkWin = ostypes.TYPE.GdkWindow.ptr(ctypes.UInt64(winPtrStr));
+  const gtkWin = ostypes.HELPER.gdkWinPtrToGtkWinPtr(gdkWin);
+  ostypes.API.gtk_window_set_keep_above(gtkWin, 1);
```

Comment:
`JPM [error]   Message: TypeError: ostypes.API.gtk_window_set_keep_above is not a function`

Fixed with `ostypes.API('gtk_window_set_keep_above')(gtkWin, 1)`. Topify now works on gtk3 :+1: